code/processes/tailsort.q

```diff
@@ -42,6 +43,13 @@ addpattr:{[hdbdir;pt;tabname]
   ];
   };
 
+/- notify rdb when tail sort process complete
+resetrdbwindow:{
+  .lg.o[`rdbwindow;"resetting rdb moving time window"];
+  rdbprocs:.servers.getservers[`proctype;.ts.rdbtypes;()!();1b;0b];
+  {neg[x]".rdb.tailsortcomplete:1b";neg[x][]}each exec w from rdbprocs;
```

Comment:
why are we blocking on the handles? we don't require a response, do we have to wait for the first rdb before we send to second etc.?

Reply:
you're right, we don't need to wait for the first rdb, will change this